CMakeLists.txt

```diff
@@ -16,13 +16,25 @@ include(FairMQLib)
 get_git_version()
 
 project(FairMQ VERSION ${PROJECT_VERSION} LANGUAGES CXX)
-message(STATUS "${BWhite}${PROJECT_NAME}${CR} ${PROJECT_GIT_VERSION} from ${PROJECT_DATE}")
 
-set(PROJECT_MIN_CXX_STANDARD 17)
+include(CTest)
+include(BundlePackages)
+
+set(PROJECT_FairCMakeModules_VERSION 0.2)
+find_package(FairCMakeModules ${PROJECT_FairCMakeModules_VERSION} QUIET)
```

Comment:
I assume, you're proposing this, so that the Summary has a `>= 0.2` in it?

According to [the docs](https://cmake.org/cmake/help/latest/command/find_package.html#package-file-interface-variables) config mode packages get a variable named `<PackageName>_FIND_VERSION`. So we *could* handle that in FairCMakeModules?

Maybe like this?

```cmake
if(@PROJECT_NAME@_FIND_VERSION)
  set(_@PROJECT_NAME@_REQUESTED_VERION "${FairCMakeModules_FIND_VERSION}")
endif()
```

and in FairFindPackage2.cmake:
```cmake
if(@PROJECT_NAME@_FOUND)
  list(APPEND PROJECT_PACKAGE_DEPENDENCIES @PROJECT_NAME@)
  if(_@PROJECT_NAME@_REQUESTED_VERION)
    set(PROJECT_@PROJECT_NAME@_VERSION "${_@PROJECT_NAME@_REQUESTED_VERION}")
    unset(_@PROJECT_NAME@_REQUESTED_VERION)
  endif()
endif()
```

Reply:
> I assume, you're proposing this, so that the Summary has a >= 0.2 in it?

yes

Reply:
good idea